documentation/concepts/03A_transaction_fees.md

```diff
@@ -0,0 +1,99 @@
+---
+id: transactions
+title: Transactions Fees
+sidebar_label: Transaction Fees
+---
+
+# Transactions Fees
+
+A **transaction fee** is a fee that is required in order to process a transaction on the Aleo network. Currently, there are three types of transaction fees that correspond to each transaction type: **transfer**, **deploy**, and **execute**. Transaction sizes are averages using [Big _O_ notation (on the order of)](https://en.wikipedia.org/wiki/Big_O_notation).
```

Comment:
> Transaction sizes are...
This line placement is a little unclear. Maybe add this line to the bottom Notes section?